### ISSUE TITLE
Platform download button A/B test

### ISSUE THREAD

**Issue by alexgibson** [mozilla/bedrock#9637]
## Description

(Copy/pasting from EU team trello board)

**Page**
https://www.mozilla.org/firefox/new/

**Locales**
de, en-GB, fr

**Analysis**
Almost 1% of all visitors landing on firefox/new goes to one of our other download pages, firefox/all or firefox/browsers/windows-64-bit/.
Since we automatically detect their platform we want to funnel users that want to download Firefox for the exact system and language they are on through the firefox/new page. By increasing the download rate by 0.5% points we increase the total number of downloads by 400 per day.

**Goal**
Increase the CVR on our most important download page

**Hypothesis**
If we detect the platform the user is on and display the information directly on the CTA (Download Firefox for Windows) then we will increase the download rate on this page because less visitors will go to firefox/all.

**What**
- detect the platform(=OS) of the visitor and show the value in the CTA

Version A: Control - 50% of traffic
Version B: Test - 50% of traffic
Confidencelevel: 95%
Runtime: 1 full week

**Measurements**
- Sessions
- Downloads
- Download Rate
- Bounce Rate

**Localization**
DE
'Firefox für Windows herunterladen'
'Firefox für macOS herunterladen'
'Firefox für Linux herunterladen'

FR
'Télécharger Firefox pour Windows'
'Télécharger Firefox pour macOS'
'Télécharger Firefox pour Linux'

**Comment by alexgibson** [comment]
The following three experiments are ready for code review and QA on stage:

- https://app.convert.com/#/projects/1003343/summary/100310775
- https://app.convert.com/#/projects/1003343/summary/100310769
- https://app.convert.com/#/projects/1003343/summary/100310689

Note: not yet ready to launch on prod (we're waiting on % for each variation from the data science team).

**Comment by alexgibson** [comment]
QA checklist:

- Clicking any download button on the page registers as a primary goal.
- Clicking the "Download options and other languages" link registers as a secondary goal.

**Comment by slightlyoffbeat** [comment]
Tested on Mac & browserstack for Windows.
- primary & secondary goals are triggering as expected
- text is inserted into each link
- turned each test on and tested on staging in different browsers.

two notes:
- I don't believe I can test the 4th item in your checklist until traffic redirects are in place.
- I noticed that confidence level needs to be set to 95% for each test (currently 97%)

Otherwise, this all seems good to me.
